### DISCUSSION TITLE
Add a "proper" history card.

### DISCUSSION THREAD

**Discussion by theorician** [home-assistant/frontend#8133] [Dashboard]
Now that we have a logbook card, it would be great if we could have a history card as well. That is, one where I can choose which elements I want the history for - like in the existing history graph card - and then also choose which date and time period I want to see the history for.

The current history graph card only shows the last n days, a period which is also fixed. The current history tab lets you choose which date and time range to pick.
The current history tab lets you pick a single entity or all entities, but not a number of entities. The history graph card lets you choose any number of entities to include.
The history graph card is, well, a card. I can put it where I want it and choose exactly what gets displayed. The history tab gives users a view of _everything_ that's going on and its location is static.

It'd be great if we had a card that got the best of all these features.

**Comment by mrkeuz** [comment]
Also, it would be good to add some "Global" time choose control for all history cards on the page since it is done in Grafana or Kibana UI. So when you change this global "range" control - all cards will be redrawn with this time range.

![2021-01-29__18-49-44](https://user-images.githubusercontent.com/6247921/106296774-21021100-6263-11eb-8ea1-cd2d35b38e53.png)

![2021-01-29__18-51-39](https://user-images.githubusercontent.com/6247921/106296794-25c6c500-6263-11eb-9332-65b552108b48.png)
